gapic-generator/lib/google/gapic/generators/base_generator.rb

```diff
@@ -15,9 +15,12 @@
 # limitations under the License.
 
 require "action_controller"
-require "protobuf/descriptors"
 require "tempfile"
 
+$LOAD_PATH.unshift ::File.expand_path("../../../../../vendor", __FILE__)
```

Comment:
Can we move this load path hack to only be on schema/api and then load schema/api everywhere we need this?

Reply:
I'm a bit unsure of how you wanted it without introducing a load cycle so I isolated the load path to loader.rb.

Reply:
I simply added the generated files to the lib.